os/arch/arm/src/s5j/s5j_i2s.h

```diff
@@ -111,7 +111,7 @@ extern "C" {
  *
  ****************************************************************************/
 
-FAR struct i2s_dev_s *s5j_i2s_initialize(void);
+FAR struct i2s_dev_s *s5j_i2s_initialize(int);
```

Comment:
You shoud add the argument name. Don't add type only.

Reply:
Changed the argument type to uint16_t and added argument name.